mne/preprocessing/ica.py

```diff
@@ -144,36 +161,57 @@ class ICA(ContainsMixin):
     Parameters
     ----------
     n_components : int | float | None
-        The number of components used for ICA decomposition. If int, it must be
-        smaller than `max_pca_components`. If None, all PCA components will
-        be used. If float between 0 and 1, components will be selected by the
-        cumulative percentage of explained variance.
+        Controls the number of PCA components from the pre-ICA PCA entering the
+        ICA decomposition in the `fit()` method.
```

Comment:
One option would be to link with
```
:meth:`ICA.fit`
```
Or maybe even this would work (you can test locally with `make html_dev-noplot` in the `doc/` directory)
```
:meth:`fit`
```